lib/components/command_bar.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:provider/provider.dart';
+import 'package:flutter/material.dart';
+import 'package:rtchat/models/commands.dart';
+import 'package:rtchat/models/channels.dart';
+import 'package:rtchat/models/adapters/actions.dart';
+
+class CommandBarWidget extends StatelessWidget {
+  final void Function() chatInputUnfocus;
+  static const _commandBarHeight = 55.0;
+
+  const CommandBarWidget({Key? key, required this.chatInputUnfocus})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: _commandBarHeight,
+      child: Scrollbar(
+        isAlwaysShown: false,
+        child: ListView(
```

Comment:
I can see why you chose to use `Provider` down below, but that actually results in a bug: if the model changes without keyboard reopening, the change won't be reflected here. Instead, wrap the ListView in a Consumer and construct the items directly:

```
child: Consumer<CommandsModel>(builder: (context, commandsModel, child) {
  return ListView.builder(
    itemCount: commandsModel.commands.length,
    itemBuilder: (context, index) => TextButton(
      child: Text(commandsModel.commands[i]),
      onPressed: () { ... },
    ),
  );
}),